.github/workflows/eval-regression.yaml

```
@@ -13,36 +13,24 @@ permissions:
 jobs:
   llm_evals:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || (secrets.BRAINTRUST_API_KEY != null && (secrets.AZURE_API_KEY != null || secrets.AWS_BEARER_TOKEN_BEDROCK != null))
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)